pkg/proxy/ipvs/netlink_linux.go

```diff
@@ -173,10 +172,21 @@ func (h *netlinkHandle) GetLocalAddresses(dev, filterDev string) (sets.String, e
 		routeFilter.LinkIndex = chosenLinkIndex
 		filterMask |= netlink.RT_FILTER_OIF
 	}
+
 	routes, err := h.RouteListFiltered(netlink.FAMILY_ALL, routeFilter, filterMask)
 	if err != nil {
 		return nil, fmt.Errorf("error list route table, err: %v", err)
 	}
+
+	if h.isIPv6 {
```

Comment:
Instead of list all ipv4 and ipv6 routes, then filter out, can we only list routes that we expected?

Reply:
okay,i'll check it and modify later,thx!